src/main/ipc-main-handlers.ts

```diff
@@ -167,5 +167,5 @@ export const registerIpcMainHandlers = () => {
   ipcMain.handle(API.validatePassword, validatePassword);
   ipcMain.handle(API.createUserProfile, createUserProfile);
   ipcMain.handle(API.deleteUserProfile, deleteUserProfile);
-  ipcMain.handle(API.getFileContent, readFileSync);
+  ipcMain.handle(API.getFileContent, (_, path) => readFileSync(path));
```

Comment:
`readFileSync` accepts two arguments, so no need to call here explicitely.
```suggestion
  ipcMain.handle(API.getFileContent, readFileSync);
```